.gitignore

```diff
@@ -1,4 +1,160 @@
-*__pycache__
-.env
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
```

Comment:
@Siddhesh-Agarwal - not sure if we need all of these in the gitignore at the moment based on our current codebase

Reply:
Well, I guess we can trim it down to your original `.gitignore` + virtual environments.

Reply:
Yea that would be great if you could add a commit to fix that